resources/home/dnanexus/utils/io.py

```
@@ -0,0 +1,366 @@
+
+    headers = df.iloc[0, :].tolist()
+    missing = set(required_headers) - set(headers)
+
+    assert not missing, f"{file_id} is missing {missing} in its headers"
+
+    df.insert(0, "a", ["CNV excluded regions"] + [None] * (len(df) - 1))
+
+    return df
+
+
+def write_output_file(
```

Ruff (0.8.0)

`write_output_file` is too complex (28 > 10)

(C901)